src/capi/azext_capi/helpers/logger.py

```diff
@@ -6,8 +6,8 @@
 # pylint: disable=missing-docstring
 
 import logging
-from functools import lru_cache
 
+from functools import lru_cache
```

Comment:
This import should stay grouped with `import logging`, because we're trying to follow good old [PEP8](https://peps.python.org/pep-0008/#imports) conventions.